tests/Tests/500__jupyterhub/test-pipChanges-not-permanent.robot

```diff
@@ -0,0 +1,29 @@
+*** Settings ***
+Resource         ../../Resources/Page/ODH/JupyterHub/ODHJupyterhub.resource
+Resource         ../../Resources/Common.robot
+Suite Setup      Dashboard Test Setup
+Suite Teardown   End Web Test
+
+
+*** Test Cases ***
+Verify pip changes after Restarting the notebook is not permenant
+  [Tags]  Sanity
+  ...     ODS-257
+  Install and Import Package In JupyterLab  paramiko
+  Stop JupyterLab Notebook Server
+  Capture Page Screenshot
+  Fix Spawner Status
+  Spawn Notebook With Arguments
+  Capture Page Screenshot
+  Add and Run JupyterLab Code Cell in Active Notebook  import paramiko
+  Wait Until JupyterLab Code Cell Is Not Active
```

Comment:
I'd remove from here and move to the `Verify package is Not Installed In JupyterLab` keyword

Reply:
ack  did the changes